packages/icons/src/index.js

```diff
@@ -113,7 +118,6 @@ export { default as search } from './library/search';
 export { default as separator } from './library/separator';
 export { default as share } from './library/share';
 export { default as shortcode } from './library/shortcode';
-export { default as star } from './library/star';
```

Comment:
Was this icon ever published as part of a stable WordPress or Gutenberg release?

If so, we can't simply remove it, since there's a backwards-compatibility commitment.

If it's only ever been published in the plugin, we can deprecate it after two versions.

See:

- https://developer.wordpress.org/block-editor/developers/backward-compatibility/deprecations/
- https://github.com/WordPress/gutenberg/tree/master/packages/deprecated

If it was published in a stable WordPress release, then it must be kept forever.

Reply:
Hm, or is this a unique package in how it's not actually enqueued in the editor? I guess at the very least, if it's only ever intended to be consumed as a package, we'd still want to include a CHANGELOG entry about the removal being a breaking change.

See: https://github.com/WordPress/gutenberg/blob/master/packages/README.md#maintaining-changelogs

Reply:
It was added, as an oversight [here](https://github.com/WordPress/gutenberg/pull/21209/). `star-filled` already existed (and was identical in it's intended application), so `star` is extraneous. Let me know how you think we should proceed.

Reply:
I guess if it isn't exposed directly on the `window` global, and it never landed in a published version of the package ([verified](https://unpkg.com/browse/@wordpress/icons@1.3.1/src/index.js)), then no additional action should be necessary.